superset/commands/database/ssh_tunnel/create.py

```diff
@@ -39,15 +40,10 @@ def __init__(self, database: Database, data: dict[str, Any]):
         self._properties = data.copy()
         self._properties["database"] = database
 
+    @transaction(on_error=partial(on_error, reraise=SSHTunnelCreateFailedError))
     def run(self) -> Model:
-        try:
-            self.validate()
-            ssh_tunnel = SSHTunnelDAO.create(attributes=self._properties, commit=False)
```

Comment:
Seems kind of weird how the command didn't actually commit. I presume the intent was the command was to be used in association with other commands.